File: frontend/src/app/shared/models/customer.model.ts
export class Customer {
  id: string;
  firstName: string;
  lastName: string;
  phone: string;
  email: string;
  // region: string;
  // city: string;

  get name(): string {
    return `${this.firstName} ${this.lastName}`;
  }

  // get nameWithAddress(): string {
  //   return `${this.firstName} ${this.lastName} (${this.city})`;
  // }

  get nameWithPhone(): string {
    return `${this.firstName} ${this.lastName} (${this.phone})`;
  }

  constructor(data: any) {
    Object.assign(this, data);
  }
}

File: Mushka.WebApi/ClientModels/Customer/CustomerModel.cs
using System;

namespace Mushka.WebApi.ClientModels.Customer
{
    public class CustomerModel
    {
        public Guid Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Email { get; set; }

        public string Phone { get; set; }

        public string City { get; set; }

        public string Region { get; set; }
    }
}